src/test/scala/nl.knaw.dans.easy.bagstore/component/PruneSpec.scala

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.bagstore.component
+
+import java.io.ByteArrayInputStream
+import java.nio.file.Paths
+import java.util.UUID
+
+import better.files.File
+import nl.knaw.dans.bag.v0.DansV0Bag
+import nl.knaw.dans.easy.bagstore.BagId
+import nl.knaw.dans.lib.error._
+
+import scala.util.{ Failure, Success }
+
+class PruneSpec extends BagProcessingFixture {
+
+  implicit val referencedStore: BagPath = store1
+
+  "prune" should "not prune when a file in one is a folder in the other and vice versa" in {
+    val uuid = UUID.randomUUID()
+    DansV0Bag.empty(File(store1) / pathFromUUID(uuid) / "bag").getOrRecover(e => fail(e))
```

Comment:
replace all `e => fail(e)` with `fail`

Reply:
That does not compile but `fail(_)` does.